app/scripts/test-e2e-local.sh

```
@@ -0,0 +1,528 @@
+    export IOS_SIMULATOR_NAME="$SIMULATOR_NAME"
+
+    echo "Simulator status:"
+    xcrun simctl list devices | grep "$AVAILABLE_SIMULATOR"
+}
+
+build_ios_app() {
+    log_info "🔨 Building iOS app..."
+    # Set environment variable for e2e testing to enable OpenSSL fixes
+    export E2E_TESTING=1
+
+    if ! xcodebuild -workspace ios/OpenPassport.xcworkspace -scheme OpenPassport -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build -jobs $(sysctl -n hw.ncpu) -parallelizeTargets SWIFT_ACTIVE_COMPILATION_CONDITIONS="DEBUG E2E_TESTING"; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)